bigquery/google/cloud/bigquery_v2/proto/model.proto

```diff
@@ -514,103 +520,105 @@ message Model {
   }
 
   // Output only. A hash of this resource.
-  string etag = 1;
+  string etag = 1 [(google.api.field_behavior) = OUTPUT_ONLY];
 
   // Required. Unique identifier for this model.
-  ModelReference model_reference = 2;
+  ModelReference model_reference = 2 [(google.api.field_behavior) = REQUIRED];
 
-  // Output only. The time when this model was created, in millisecs since the
-  // epoch.
-  int64 creation_time = 5;
+  // Output only. The time when this model was created, in millisecs since the epoch.
+  int64 creation_time = 5 [(google.api.field_behavior) = OUTPUT_ONLY];
 
-  // Output only. The time when this model was last modified, in millisecs
-  // since the epoch.
-  int64 last_modified_time = 6;
+  // Output only. The time when this model was last modified, in millisecs since the epoch.
+  int64 last_modified_time = 6 [(google.api.field_behavior) = OUTPUT_ONLY];
 
-  // [Optional] A user-friendly description of this model.
-  string description = 12;
+  // Optional. A user-friendly description of this model.
+  string description = 12 [(google.api.field_behavior) = OPTIONAL];
 
-  // [Optional] A descriptive name for this model.
-  string friendly_name = 14;
+  // Optional. A descriptive name for this model.
+  string friendly_name = 14 [(google.api.field_behavior) = OPTIONAL];
 
-  // [Optional] The labels associated with this model. You can use these to
-  // organize and group your models. Label keys and values can be no longer
+  // The labels associated with this model. You can use these to organize
+  // and group your models. Label keys and values can be no longer
   // than 63 characters, can only contain lowercase letters, numeric
   // characters, underscores and dashes. International characters are allowed.
   // Label values are optional. Label keys must start with a letter and each
   // label in the list must have a different key.
   map<string, string> labels = 15;
```

Comment:
Hm, I think the ` [(google.api.field_behavior) = OPTIONAL]` annotation may have been missed for labels? I'll check with someone from ACTools